### PR TITLE
Update text for email/feedback

### LINKED ISSUE #198: Renderings popup window: feedback text updates
Suggest that current text at bottom of renderings popup window that is currently:

> How did you like these interpretations? Give us feedback or report issues by emailing image@cim.mcgill.ca. Feel free to send us a link to the graphic you were using , but make sure you do not send any personal information.
>
> Alternatively, you can use our feedback form.

be changed to:

> How did you like these interpretations? We love getting feedback and bug reports using our feedback form. For anything else, you can also email us at image@cim.mcgill.ca. In either case, please make sure you do not send any personal information.

For two reasons:
1. I think we want to prioritize the feedback form, and encourage people to use that if at all possible. The email is really a fallback.
2. I think that is a bit more light/friendly, and a bit more concise

### PR DESCRIPTION
**Pull request by JRegimbal**
Resolve #198. Minor changes really only adjusting text, although now we don't insert any HTML elements for the page.

### CODE REVIEW

**Comment by jeffbl** [comment]
@Cybernide @jaydeepsingh25 This is pending your approval before merge, so did not make it in to the extension submission last night. I'm putting this on a list of items for another submission before CSUN, perhaps Saturday morning if possible, assuming the previous submission gets approved by then!

File: src/info/info.ts
Comment (jaydeepsingh25) [inline src/info/info.ts:500]: not blocking, but may be you can also remove feedbackLinkText from messages.json.

**Review by jaydeepsingh25 — APPROVED** [review]

**Pull request merged**
